VMEncryption/main/handle.py

```diff
@@ -1011,15 +1011,18 @@ def enable_encryption():
                 public_settings = get_public_settings()
                 confidential_encryption_tempdisk = public_settings.get("PrivatePreview.ConfidentialEncryptionTempDisk")
                 encryptResourceDiskforCVM = False
+                encryptResourceDisk = False
```

Comment:
Two flags seems redundant here. Can we just use encryptResourceDisk flag and set it to true either for CVM or EFA?